boards/arm/rak4631_nrf52840/rak4631_nrf52840.dts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2020 Guillaume Paquet <guillaume.paquet@smile.fr>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include <dt-bindings/lora/sx126x.h>
+
+/ {
+	model = "RAKWireless RAK4631 WisBlock LPWAN Module with a Nordic NRF52840 SoC";
+	compatible = "nordic,rak4631_nrf52840";
+
+	chosen {
+		zephyr,console = &uart1;
+		zephyr,shell-uart = &uart1;
+		zephyr,uart-mcumgr = &uart1;
+		zephyr,bt-mon-uart = &uart1;
+		zephyr,bt-c2h-uart = &uart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_2 {
+			gpios = <&gpio1 4 GPIO_ACTIVE_LOW>;
+			label = "Blue LED";
+		};
+		green_led: led_1 {
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+			label = "Green LED";
+		};
+	};
+
+	/* Declaration of aliases */
+	aliases {
+		led0 = &blue_led;
+	};
+};
+
+&adc {
+	status = "okay";
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <20>;
+	rx-pin = <19>;
+};
+
+&uart1 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "okay";
+	tx-pin = <16>;
+	rx-pin = <15>;
+};
+
+&i2c1 {
+	compatible = "nordic,nrf-twi";
+	status = "okay";
+	sda-gpios = <&gpio0 24 0>;
+	scl-gpios = <&gpio0 25 0>;
+};
+
+&spi1 {
+	compatible = "nordic,nrf-spi";
+	status = "okay";
+	sck-pin = <43>;
+	mosi-pin = <44>;
+	miso-pin = <45>;
+	cs-gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+
+	lora: sx1262@0 {
+		compatible = "semtech,sx1262";
+		reg = <0>;
+		label = "sx1262";
+		reset-gpios = <&gpio0 19 GPIO_ACTIVE_LOW>;
+		busy-gpios = <&gpio0 27 GPIO_ACTIVE_LOW>;
+		tx-enable-gpios = <&gpio1 7 GPIO_ACTIVE_LOW>;
+		rx-enable-gpios = <&gpio1 5 GPIO_ACTIVE_LOW>;
+		/* antenna-enable-gpios = <&gpio0 1 GPIO_ACTIVE_HIGH>; */
```

Comment:
@gpaquet85 I guess this line can be removed since it won't be used

Reply:
yes I am asking internal schematic to RAKWireless to be sure we can remove it

Reply:
@gpaquet85 sounds good.